app/account/page.tsx

```
@@ -279,7 +301,7 @@ export default function AccountPage() {
                     })}
                   </div>
                 </div>
-
+                //TODO: Fetch actual listings count from the database.
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)